### PR TITLE
FUSETOOLS-2465: Update the latest SAP version for unknown versions

### PR DESCRIPTION
**Pull request by apodhrad**
Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

### CODE REVIEW

File: jboss-fuse-sap-tool-suite/uitests/plugins/org.jboss.tools.fuse.sap.reddeer/src/org/jboss/tools/fuse/sap/reddeer/SupportedSAPVersions.java [hunk 1]
```diff
@@ -50,7 +54,10 @@ public static String getSAPVersion(String camelVersion) {
 		case CAMEL_2_17_0_REDHAT_630262:
 			return SAP_630_REDHAT_262;
 		default:
-			return SAP_630_REDHAT_262;
+			if (camelVersion.startsWith("6.2.1")) {
```
Comment (lhein) [inline jboss-fuse-sap-tool-suite/uitests/plugins/org.jboss.tools.fuse.sap.reddeer/src/org/jboss/tools/fuse/sap/reddeer/SupportedSAPVersions.java]: Camel version 6.2.1?

Comment (apodhrad) [reply jboss-fuse-sap-tool-suite/uitests/plugins/org.jboss.tools.fuse.sap.reddeer/src/org/jboss/tools/fuse/sap/reddeer/SupportedSAPVersions.java]: Good point! I will fix it asap

**Comment by apodhrad** [comment]
fixed

**Comment by apodhrad** [comment]
@lhein do you think it is ready for merging?

**Comment by apodhrad** [comment]
testPR

**Review by lhein — APPROVED** [review]

**Pull request merged**